docker-compose.yml

```diff
@@ -1,19 +1,41 @@
 version: '3.1'
 
 services:
-  # Dependencies for running unit tests
+  # Dependencies for running unit tests and for running the 'irma' command line tool using one of the test configurations.
   postgres:
     image: postgres:13-buster
     environment:
       POSTGRES_USER: testuser
       POSTGRES_PASSWORD: testpassword
       POSTGRES_DB: test
+    networks:
+      - irma-net
     ports:
       - 5432:5432
+  postgres-init:
+    image: postgres:13-buster
```

Comment:
It would be good to upgrade to the new stable Debian release `bullseye`. That can be done later, because otherwise I would have to open a third Docker PR :)